tools/slang-embed/slang-embed.cpp

```diff
@@ -91,7 +98,7 @@ struct App
 
         if (!inputPath || (argc != 0))
         {
-            fprintf(stderr, "usage: %s inputPath [outputPath]\n", appName);
+            fprintf(stderr, "usage: %s inputPath includeDir [outputPath]\n", appName);
```

Comment:
If includeDir is required, maybe add "|| !includeDir" to the condition above, otherwise if includeDir is optional, it should be shown in brackets here.

Reply:
Uh-oh, I think auto-merge merged it in before I could react to the comment..

Let me open another PR for the fix